webapp/actions/user_actions.jsx

```diff
@@ -225,6 +225,13 @@ function populateDMChannelsWithProfiles(userIds) {
     }
 }
 
+function populateChannelWithProfiles(channelId, userIds) {
```

Comment:
We mutate the store all over the place in actions as it is and this is very similar to an existing function. While I agree that we should probably use the dispatcher for all changes to the store, we definitely don't follow that so I'm not inclined to put in the extra overhead code to do it here